openapi/openapi/src/main/scala/endpoints/openapi/JsonSchemas.scala

```diff
@@ -282,19 +338,28 @@ trait JsonSchemas extends algebra.JsonSchemas with TuplesSchemas {
         .orFallbackToJsonSchema(schemaA.ujsonSchema, schemaB.ujsonSchema),
       (schemaA.docs, schemaB.docs) match {
         case (
-            OneOf(alternatives1, maybeExample1),
-            OneOf(alternatives2, maybeExample2)
+            OneOf(alternatives1, _, maybeExample1, _),
+            OneOf(alternatives2, _, maybeExample2, _)
             ) =>
           OneOf(
             alternatives1 ++ alternatives2,
-            maybeExample1.orElse(maybeExample2)
+            example = maybeExample1.orElse(maybeExample2)
```

Comment:
I see that we don’t treat `title` and `description` in the same way as we treat `example`.

Now I’m thinking that maybe it’s better to not move the example from the alternative to the `OneOf` (like you do with `title` and `description`). What do you think?

Reply:
I think the behaviour for `example` is fine: an example for a variant is always going to also be an example for the `oneOf`. In fact, for `oneOf`, any example is always going to have to choose one specific alternative.

That's obviously not the case for a title or description (in fact, I'm struggling to imagine a case where a variant's title/description could generalize...).

Reply:
OK. Let’s keep these things as they are for now!